### PR TITLE
Obstacle Level Builder

### PR DESCRIPTION
**Pull request by OlafSzmidt**

### CODE REVIEW

File: UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/ObstacleMenu.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+using UnityEngine;
+
+public class ObstacleMenu : IMenu
+{
+    private GUIContent obstaclePickerButtonTitle = new GUIContent("View Obstacles");
+
+    public void Display()
+    {
+        DisplayTitle();
+        DisplayObstaclesPickerButton();
```
Comment (CelineBoudier) [inline UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/ObstacleMenu.cs]: aren't we listening for the picker event? or is it not part of the deal now just visualising obstacle? :thinking:

Comment (OlafSzmidt) [reply UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/ObstacleMenu.cs]: 👍

Comment (CelineBoudier) [reply UnityProject/Assets/Editor/LevelBuilder/GUI/Menus/ObstacleMenu.cs]: I WIN THE INTERNET

**Comment by OlafSzmidt** [comment]
@mrniket @CelineBoudier

**Pull request merged**